documentation/polymer-templates/tutorial-template-basic.asciidoc

```diff
@@ -8,20 +8,16 @@ layout: page
 
 In this section, we demonstrate how to create a simple component using only the Template API.
 
-It is recommended to use LitElement template language for your components, but PolymerTemplate is also supported.
-
 Our example creates a simple view that allows the user to enter their name and click a button.
 
 [NOTE]
 This documentation covers LitElement, Polymer 3 and npm in Vaadin version 14+ only. Earlier Vaadin versions support Polymer 2, HTML imports, and the Bower package manager. You can still use these components in Vaadin 14, if you work in the special *compatibility mode*. See the https://vaadin.com/docs/v13/flow/polymer-templates/tutorial-template-basic.html[Vaadin 13 documentation] for how to build client-side components with Polymer 2 and HTML imports.
 
 == Creating the Template File on the Client Side
 
-=== LitElement
-
-The first step is to create the LitElement JavaScript template file on the client side in `frontend/src/hello-world.js`. This file contains the view structure.
+The first step is to create the LitElement TypeScript template file on the client side in `frontend/src/hello-world.ts`. This file contains the view structure.
 
-*Example*: Creating the `hello-world.js` JavaScript LitElement template file.
+*Example*: Creating the `hello-world.ts` TypeScript LitElement template file.
 
 [source,js]
```

Comment:
Now that the preceding title says `TypeScript`, it would be nice to have the example code in TypeScript accordingly. Note that this is code still `[source,js]`.

```suggestion
[source,typescript]
```